platformio.ini

```diff
@@ -122,7 +122,8 @@ lib_ignore =
 build_flags =
 lib_deps =
     ZumoOta
-    https://github.com/me-no-dev/ESPAsyncWebServer.git
+   
+   https://github.com/BlueAndi/ESPAsyncWebServer.git

```

Comment:
wasn't this tilde2 branch from requirement ?

Reply:
It's the same link...